### PR TITLE
Fix/typo and match card link

### PR DESCRIPTION
**Pull request by lawandothman**

### CODE REVIEW

File: src/components/MatchCard.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const MatchCard = ({
 }) => {
   if (mini) {
     return (
-      <Card p={'1em'} w={'10em'} textAlign={'center'}>
```
Comment (pedsm) [inline src/components/MatchCard.tsx:17]: This is breaking scaling
Screenshot 2022-11-23 at 00 11 13
Everything else looks good

**Review by pedsm — CHANGES_REQUESTED** [review]

File: src/components/MatchCard.tsx (hunk 1 above)
Comment (pedsm) [reply src/components/MatchCard.tsx:17]: Nvm that was already broken but also  I kinda like the consistent icons vs full width, a day with few matches will causes cards to be too fat

Comment (lawandothman) [reply src/components/MatchCard.tsx:17]: I mean looks fine to me if this is the semi-finals, can set a maxW on the card instead, wdyt ?
image

File: src/components/MatchCard.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const MatchCard = ({
...
+      <Card p={'1em'} w='full' textAlign={'center'}>
```
Comment (pedsm) [inline src/components/MatchCard.tsx:17]:
```suggestion
      <Card p={'1em'} w={'10em'} textAlign={'center'}>
```

**Review by pedsm — APPROVED** [review]

**Pull request merged**